src/main/java/uk/gov/hmcts/divorce/divorcecase/validation/ValidationUtil.java

```diff
@@ -152,11 +149,7 @@ public static List<String> validateCasesAcceptedToListForHearing(BulkActionCaseD
 
     private static boolean isLessThanOneYearAgo(LocalDate date) {
         return !date.isAfter(LocalDate.now())
```

Comment:
I believe this test is redundant? For example date = 01/08/2024 (future date) will always be true for `date.isAfter(LocalDate.now().minusYears(1).minusDays(1))`?

Reply:
I was just trying to make sonar behave itself tbh